tests/bootstrap.php

```diff
@@ -6,64 +6,32 @@
  * @subpackage Tests
  */
 
-// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
-const WP_TESTS_PHPUNIT_POLYFILLS_PATH = __DIR__ . '/../vendor/yoast/phpunit-polyfills';
-
-// Load Core's test suite.
-$elasticsearch_extensions_tests_dir = getenv( 'WP_TESTS_DIR' );
-if ( ! $elasticsearch_extensions_tests_dir ) {
-	$elasticsearch_extensions_tests_dir = '/tmp/wordpress-tests-lib';
-}
-
-require_once $elasticsearch_extensions_tests_dir . '/includes/functions.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.UsingVariable
-
-/**
- * Setup our environment.
- */
-function elasticsearch_extensions_manually_load_environment() {
-	/*
-	 * Tests won't start until the uploads directory is scanned, so use the
-	 * lightweight directory from the test install.
-	 *
-	 * @see https://core.trac.wordpress.org/changeset/29120.
-	 */
-	add_filter(
-		'pre_option_upload_path',
-		function () {
-			return ABSPATH . 'wp-content/uploads';
+define( 'ELASTICSEARCH_EXTENSIONS_TESTS_DIR', dirname( __DIR__ ) . '/tests/' );
```

Comment:
This is really just `__DIR__`. Rather than establishing a separate constant here, I'd rather just use `__DIR__` below.